packages/cli/src/Command/Console.php

```diff
@@ -0,0 +1,225 @@
+        'release' => 'release',
+        'version' => 'version',
+        'copyright' => 'copyright',
+    ];
+
+    /**
+     * Maps all Settings.cfg sections that are not covered by this converter
+     */
+    private const MAPPING_DEPRECATED_SECTIONS = [
+        'notify',
+        'latex_elements'
+    ];
+
+    /**
+     * Maps all Settings.cfg sections that are converted
+     */
+    private const MAPPING_ACCEPTED_SECTIONS = [
+        'html_theme_options',
+        'general',
+        'intersphinx_mapping'
+    ];
+
+    public function __construct(
+    ) {
+        parent::__construct('console');
+
+        $this->addArgument(
+            'input',
+            InputArgument::REQUIRED,
+            'Path to directory where settings.cfg is stored, and guides.xml will be generated.',
+        );
+
+        $this->addOption(
+            'force',
+            null,
+            InputOption::VALUE_NONE,
+            'When set, forces to overwrite possibly existing XML.'
+        );
+
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $settingsFile = $input->getArgument('input') . '/Settings.cfg';
+        $guidesFile = $input->getArgument('input') . '/guides.xml';
+
+        if (!file_exists($settingsFile)) {
+            $output->writeln('<error>Could not open ' . $settingsFile . '</error>');
+            return Command::FAILURE;
+        }
+
+        if (file_exists($guidesFile) && !$input->getOption('force')) {
+            $output->writeln('<error>Target file already exists in specified directory (' . $guidesFile . ')</error>');
+            return Command::FAILURE;
+        }
+
+        $output->writeln('Converting ' . $settingsFile . ' to ' . $guidesFile . ' ...');
+
+        if (!$this->convertSettingsToGuide($output, $settingsFile, $guidesFile)) {
+            $output->writeln('<error>Settings could not be converted. Please check for proper syntax.</error>');
+            return Command::FAILURE;
+        }
+
+        $output->writeln('Settings converted. You can now delete Settings.cfg and add guides.xml to your repository.');
+
+        return Command::SUCCESS;
+    }
+
+    protected function convertSettingsToGuide(OutputInterface $output, string $inputFile, string $outputFile): bool
+    {
+        // Settings.cfg can be parsed as an INI file. If it fails, bail out.
+        try {
+            $settings = @parse_ini_file($inputFile, true, INI_SCANNER_RAW);
+        } catch (\Exception $e) {
+            return false;
+        }
+
+        if (!is_array($settings)) {
+            return false;
+        }
+
+        // This array will hold all setting values that we could not migrate to guides.xml
+        $unmigratedSettings = $settings;
+
+        // Holds number of converted settings
+        $convertedSettings = 0;
+
+        // Setup basic DOM Document
+        $dom = new \DOMDocument("1.0", "UTF-8");
+        $dom->preserveWhiteSpace = false;
```

Comment:
That makes reading the XML file by humans harder, as all tags are in one line?

Reply:
Changed this to `true` - however, the XML document is created from a empty DomDocument and not reading in an existing XML file, so there's no real whitespace to preserve.